src/main/java/App.java

```diff
@@ -6,6 +6,7 @@ public String getGreeting() {
         return "Hello world.";
     }
 
+    public String redundantVfar;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make redundantVfar a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)